taccsite_cms/settings.py

```diff
@@ -288,9 +287,14 @@ def gettext(s): return s
 # TACC: PORTAL
 ########################
 
-INCLUDES_CORE_PORTAL = True
-INCLUDES_PORTAL_NAV = True
-INCLUDES_SEARCH_BAR = True
+PORTAL_IS_CORE = True
```

Comment:
See comment above on `AUTH_USES_CORE` idea.

Reply:
Responded. I will remember from the prior discussion to update this instance as we decide.